src/icons/Manage.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react'
+import Svg, { Path } from 'react-native-svg'
+import colors from 'src/styles/colors'
+
+interface Props {
+  color?: string
+  testID?: string
+  size?: number
+}
+
+function Manage({ color = colors.primary, testID = 'Manage', size = 24 }: Props) {
```

Comment:
any reason to use size 24 here? the design just includes an icon of size 20 right?

Reply:
updated in [cd2b56a](https://github.com/valora-inc/wallet/pull/5977/commits/cd2b56acba08660492020bb1b5764cfc924a171f)